src/components/StoresList/StoresList.js

```diff
@@ -10,8 +10,7 @@ const StoresList = props => {
         ? props.storeList.map(store =>
             <li key={store.id}>
               <p>
-                <strong>{store.name}</strong>
-                <br />
+                <h2>{store.name}</h2>
```

Comment:
Hello... I am getting a console error here. It says "Warning: validateDOMNesting(...): 'h2' cannot appear as a descendant of 'p'.